server.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"UrlMinifier/shortener"
+	"UrlMinifier/store"
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type UrlCreationRequest struct {
+	LongUrl string `json:"long_url" binding:"required"`
+}
+
+func createShortUrl(c *gin.Context) {
```

Comment:
This is a good example for gin!

The problem with gin is that it is not http / net compatible.
This is usually needed in order to increase performance (although in 99% of cases the bottleneck will be in the database)

Let's take chi or the net / http standard library for the tutorial.